docs/docs/styled-components.md

```diff
@@ -112,7 +112,29 @@ export default () => (
 )
 ```
 
-### Enabling user stylesheets with a stable class name
+## Creating Global Styles
+
+Styled-components are primarily used for a single CSS class that is isolated from other components. In some cases, you want to override global styling — for example, the default margins of your `body` element. Styled-components has your back. You can use the `createGlobalStyle` to accomplish this. We advise using `createGlobalStyle` in Layout components, which are shared over multiple pages rather than using it on a single page.
+
+The example below shows how to create a `GlobalStyle` (which is a StyledComponent) for the color purple by importing `createGlobalStyle` from `styled-components`.
+
+```jsx:title=src/pages/index.js
+import { createGlobalStyle } from 'styled-components'
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    color: ${props => (props.theme === 'purple' ? 'purple' : 'white')};
+  }
+`
+
+// later on in your file
+
```

Comment:
```suggestion
export default ({children}) => (
```
do we need to mention props here, such as initializing a `theme` object prop with a default value like this?
```
export default ({children, theme = {}}) => (
```

Reply:
@marcysutton I think you're referring to global theming like theme-ui supports. This is a "local" theming by using a prop called theme which in this example is just a string. Maybe we should rename it to a prop called `variation`?

Reply:
I think we can keep it simple as a string, thanks for the input @wardpeet.